app/assets/scripts/components/explore/Summary.js

```diff
@@ -118,32 +118,34 @@ class Summary extends Component {
                   PDF Report
                 </a>
               </li>
-              <li>
-                <a
-                  href={model.sourceData && model.sourceData.clusters}
-                  target='_blank'
-                  rel='noopener noreferrer'
-                  className={c('drop__menu-item drop__menu-item--shapefile',
-                    { disabled: !model.sourceData || !model.sourceData.clusters }
-                  )}
-                  data-hook='dropdown:close'
-                >
-                  Country clusters
-                </a>
-              </li>
-              <li>
-                <a
-                  href={model.sourceData && model.sourceData.scenarios}
-                  target='_blank'
-                  rel='noopener noreferrer'
-                  className={c('drop__menu-item drop__menu-item--data',
-                    { disabled: !model.sourceData || !model.sourceData.scenarios }
-                  )}
-                  data-hook='dropdown:close'
-                >
-                  Full scenario data
-                </a>
-              </li>
+              {
+                model.sourceData && model.sourceData.clusters &&
+                  <li>
+                    <a
+                      href={model.sourceData && model.sourceData.clusters}
```

Comment:
The conditional is not needed since it is already done above

```suggestion
                      href={model.sourceData.clusters}
```